### ISSUE TITLE
Support for high level panels

### ISSUE THREAD

**Issue by metalmatze** [perses/perses#200]
It would be great to support high-level panels, that consist of a set of panels. 
Essentially grouping the most common panels into one. 
One example I can immediately think of would be the RED and USE methods. 

Instead of describing 3 panels for RED, I would prefer having one panel that takes 3 expressions: total (rate), errors, duration (histogram_quantile), and then has some presets for showing these 3 panels already. I've built that at least 20x in the past.

**Comment by Nexucis** [comment]
I'm just adding more info around this issue :  RED stand for Rate, Error, Duration. 

Tell me if I'm wrong @metalmatze,  but a good representation of what you have in mind can be found here: https://demo.pyrra.dev/objectives?expr={__name__=%22parca-grpc-query-latency%22,%20namespace=%22parca%22}&grouping={}, right ?

Overall I find the feature interesting. Having a super Chart/ Config that is generating multiple sub `Chart` shouldn't be a big issue to handle. 

/cc @juliusv @LukeTillman @sjcobb

**Comment by AntoineThebaud** [comment]
Kinda related to this, Grafana allows to repeat panels & whole rows based on variables. I'd not be suprised if this feature is critical for some people.

Ref: https://grafana.com/blog/2020/06/09/learn-grafana-how-to-automatically-repeat-rows-and-panels-in-dynamic-dashboards/

**Comment by sjcobb** [comment]
Yeah we’ve noticed a ton of repeat row usage in customer dashboards, do you all use it at Amadeus @AntoineThebaud? Are there any options here besides copying Grafana, seems like to meet that use case, we’d pretty much have to 🫤

**Comment by AntoineThebaud** [comment]
yes @sjcobb we don't use it everywhere but we have a bunch of cases where row repeat & panel repeat are useful.

**Comment by AntoineThebaud** [comment]
> Kinda related to this, Grafana allows to repeat panels & whole rows based on variables. I'd not be suprised if this feature is critical for some people.
> 
> Ref: https://grafana.com/blog/2020/06/09/learn-grafana-how-to-automatically-repeat-rows-and-panels-in-dynamic-dashboards/

Since these repeat features are quite different from OP's request, I created dedicated issues for them (https://github.com/perses/perses/issues/2936 and https://github.com/perses/perses/issues/2937)

**Comment by AntoineThebaud** [comment]
I think we are in a good state to close this issue now:
- Since https://github.com/perses/plugins/pull/85 the Table panel can embed sub-panels for data rendering, and this feature was actually implemented with the RED use case in mind as you can see on the example screenshots there.
- As shared above, when going the [Dashboard-as-Code](https://perses.dev/perses/docs/dac/introduction/#dashboard-as-code) way it's quite easy to define any kind of reusable components, and e.g the RED use case could be achieved with a reusable/templated panel group that would include 3 panels - one for each metric.

Let me know if you disagree and we can always reopen this issue.
